src/Employer/Employer.Web/ViewModels/VacancyDisplayViewModel.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Humanizer;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Esfa.Recruit.Employer.Web.ViewModels
+{
+    public abstract class DisplayVacancyViewModel
+    {
+        public string ApplicationInstructions { get; internal set; }
+        public string ApplicationUrl { get; internal set; }
+        public bool CanDelete { get; internal set; }
+        public bool CanSubmit { get; internal set; }
+        public string ContactName { get; internal set; }
+        public string ContactEmail { get; internal set; }
+        public string ContactTelephone { get; internal set; }
+        public string ClosingDate { get; internal set; }
+        public string EmployerDescription { get; internal set; }
+        public string EmployerName { get; internal set; }
+        public string EmployerWebsiteUrl { get; internal set; }
+        public string ExpectedDuration { get; internal set; }
+        public string HoursPerWeek { get; internal set; }
+        public Address Location { get; internal set; }
+        public IEnumerable<string> EmployerAddressElements => new[] { Location.AddressLine1, Location.AddressLine2, Location.AddressLine3, Location.AddressLine4, Location.Postcode }
+                                                                .Where(x => !string.IsNullOrEmpty(x));
+        public string MapUrl { get; set; }
+        public int NumberOfPositions { get; internal set; }
+        public string NumberOfPositionsCaption => $"{"position".ToQuantity(NumberOfPositions)} available";
+        public string OutcomeDescription { get; internal set; }
+        public string PossibleStartDate { get; internal set; }
+        public string ProviderName { get; internal set; }
+        public string ProviderAddress { get; internal set; }
+        public List<string> Qualifications { get; internal set; }
```

Comment:
To standardise shall we change this to `IEnumerable<string> Qualifications { get; internal set; }`

Reply:
👍